src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -2,23 +2,30 @@ package org.broadinstitute.monster.encode.extraction
 
 import com.spotify.scio.ScioContext
 import com.spotify.scio.coders.Coder
+import com.spotify.scio.transforms.ScalaAsyncLookupDoFn
 import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.coders.{KvCoder, StringUtf8Coder}
+import org.apache.beam.sdk.transforms.{GroupIntoBatches, ParDo}
+import org.apache.beam.sdk.values.KV
 import org.broadinstitute.monster.common.PipelineBuilder
 import org.broadinstitute.monster.common.StorageIO.writeJsonLists
-import org.broadinstitute.monster.common.msg.MsgOps
-import org.broadinstitute.monster.common.msg.UpackMsgCoder
+import org.broadinstitute.monster.common.msg.{JsonParser, MsgOps, UpackMsgCoder}
 import upack._
 
-object ExtractionPipelineBuilder extends PipelineBuilder[Args] {
+import scala.collection.JavaConverters._
+import scala.concurrent.Future
+
+/**
+  *
+  * Builder for the ENCODE metadata extraction pipeline.
+  *
+  * @param getClient function that will produce a client which can interact with the ENCODE API
+  */
+class ExtractionPipelineBuilder(getClient: () => EncodeClient)
```

Comment:
You'll eventually want `batchSize` as an arg here too, but that can wait until the unit-testing ticket.